lisa/operating_system.py

```diff
@@ -754,7 +754,8 @@ def _initialize_package_installation(self) -> None:
         # Use below command to update rhui-microsoft-azure-rhel package from microsoft
         #  repo to resolve the issue.
         # Details please refer https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/redhat/redhat-rhui#azure-rhui-infrastructure # noqa: E501
-        if "Red Hat" == information.vendor:
+        # vendor name - Red Hat Enterprise Linux Server
+        if "Red Hat" in information.vendor:
```

Comment:
The vendor name should be "Red Hat", I tested it. Let me know, which rh version shows it as "Red Hat Enterprise Linux Server". I will fix the pattern.